sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/AppConfigurationConstants.java

```diff
@@ -32,6 +32,11 @@ public class AppConfigurationConstants {
      * Feature Store Prefix
      */
     public static final String FEATURE_STORE_SUFFIX = ".appconfig";
+    
+    /**
+     * Separator for multiple labels.
+     */
+    public static final String LABEL_SEPARATOR = ",";
```

Comment:
Where is this used?

Reply:
It should be used in `AppConfigurationStoreSelects`. Not sure why it isn't. Looks like this line was copied into that file. Will fix.